src/resources/organization_export.yaml

```diff
@@ -0,0 +1,96 @@
+---
+!include ../includes.yaml
+name: organization_export
+comment: |
+  An _organization_export_ object represents a request to export the complete data of an Organization
+  in JSON format.
+
+  To export an Organization using this API:
+
+  * Create an  and store the id that is returned.\
+  * Request the _organization_export_ every few minutes, until the _state_ field contains 'finished'.\
+  * Download the file located at the URL in the _download_url_ field.
```

Comment:
I had fun reverse-engineering the removeLineBreaks to get this bulleted list to work. Not sure how my trailing \ will work in other outputs than the reference .md, feel free to change it if it doesn't play well.